doc/api/crypto.md

```diff
@@ -2651,6 +2651,18 @@ added: v10.0.0
 Enables the FIPS compliant crypto provider in a FIPS-enabled Node.js build.
 Throws an error if FIPS mode is not available.
 
+### crypto.signOneShot(data, privateKey)
```

Comment:
IIRC all other functions that consume keys take them as the first argument.

Reply:
I ordered them according to the ordinary streaming flow (`.update(data)` followed by `.sign(privateKey)`), which made sense to me.

Reply:
I understand that and it makes sense, but it deviates from all other functions.